packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerReminderSignedOut.tsx

```diff
@@ -0,0 +1,209 @@
+import { css, ThemeProvider } from '@emotion/react';
+import { Button, buttonBrandAlt } from '@guardian/src-button';
+import { textSans } from '@guardian/src-foundations/typography';
+import { error, neutral, space } from '@guardian/src-foundations';
+import { TextInput } from '@guardian/src-text-input';
+import React from 'react';
+import { Container } from '@guardian/src-layout';
+import { BannerEnrichedReminderCta } from '../../common/types';
+import { ensureHasPreposition, ReminderStatus } from '../../../utils/reminders';
+import { useContributionsReminderEmailForm } from '../../../../hooks/useContributionsReminderEmailForm';
+import { SvgCheckmark } from '@guardian/src-icons';
+import { from } from '@guardian/src-foundations/mq';
+import { CtaSettings } from '../settings';
+import { buttonStyles } from '../buttonStyles';
+
+// ---- Component ---- //
+
+export interface MomentTemplateBannerReminderSignedOutProps {
+    reminderCta: BannerEnrichedReminderCta;
+    reminderStatus: ReminderStatus;
+    onReminderSetClick: (email: string) => void;
+    setReminderCtaSettings?: CtaSettings;
+}
+
+export function MomentTemplateBannerReminderSignedOut({
+    reminderCta,
+    reminderStatus,
+    onReminderSetClick,
+    setReminderCtaSettings,
+}: MomentTemplateBannerReminderSignedOutProps): JSX.Element {
+    return (
+        <div css={styles.container}>
+            <Container>
+                {reminderStatus !== ReminderStatus.Completed && (
+                    <Signup
+                        reminderLabel={reminderCta.reminderFields.reminderLabel}
+                        reminderStatus={reminderStatus}
+                        onSubmit={onReminderSetClick}
+                        setReminderCtaSettings={setReminderCtaSettings}
+                    />
+                )}
+
+                {reminderStatus === ReminderStatus.Completed && (
+                    <ThankYou reminderLabel={reminderCta.reminderFields.reminderLabel} />
+                )}
+            </Container>
+        </div>
+    );
+}
+
+// ---- Helper components ---- //
+
+interface SignupProps {
+    reminderLabel: string;
+    reminderStatus: ReminderStatus;
+    onSubmit: (email: string) => void;
+    setReminderCtaSettings?: CtaSettings;
+}
+
+function Signup({ reminderLabel, reminderStatus, onSubmit, setReminderCtaSettings }: SignupProps) {
+    const { email, inputError, updateEmail, handleSubmit } = useContributionsReminderEmailForm();
```

Comment:
it's great to reuse `useContributionsReminderEmailForm`, but would it be possible to share more of this file with the contributions banner?

Reply:
I'll have a look now - there are a few differences (e.g contribs one is aligned to the grid, and small copy tweaks) but I think there might be a few things we can share at least

Reply:
Okay I've tried to share as much as possible. There are some awkard differences in the `form` that I think mean it wouldn't be very elegant to try and support both, also the `header`s have slightly different copy so again probably not worth factoring out a separate component. 

I was able to pull out a shared `InfoCopy`, `ErrorCopy` and `ThankYou` components which can be shared with the contribs banner

Reply:
Great, thanks!